examples/basic/tidb-monitor.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     version: 6.1.6
   initializer:
     baseImage: pingcap/tidb-monitor-initializer
-    version: v4.0.10
```

Comment:
Just revert this, we will have dedicated PR to update the version to v5.0 later.